packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -144,18 +145,32 @@ export class RouterliciousStorageRestWrapper extends RouterliciousRestWrapper {
         const defaultQueryString = {
             token: `${fromUtf8ToBase64(tenantId)}`,
         };
-        const getAuthorizationHeader: AuthorizationHeaderGetter = async (refreshToken?: boolean): Promise<string> => {
-            // Craft credentials using tenant id and token
-            const storageToken = await tokenProvider.fetchStorageToken(
-                tenantId,
-                documentId,
-                refreshToken,
+        const getAuthorizationHeader: AuthorizationHeaderGetter = async (
+            refreshToken?: boolean
+        ): Promise<string> => {
+            return PerformanceEvent.timedExecAsync(
+                logger,
+                {
+                    eventName: "FetchStorageToken",
+                    docId: documentId,
+                    details: JSON.stringify({
+                        useRestLess,
```

Comment:
RestLess doesn't impact TokenProvider calls, so I think we should remove this to avoid incorrect correlation.

Reply:
Fair point. Removed.